util/test/sub_test.py

```diff
@@ -1420,6 +1423,30 @@ def run_compileline(flag, lookingfor):
                     do_not_test=True
                     break
 
+            elif (name == "SUPPRESSIF" and (os.access(f, os.R_OK))):
+                try:
+                    suppressme=False
+                    suppresstest=runSkipIf(f)
+                    if suppresstest.strip() != "False":
+                        suppressme = suppresstest.strip() == "True" or int(suppresstest) == 1
+                    if suppressme:
+                        suppressline = ""
+                        with open(f, 'r') as suppressfile:
+                            for line in suppressfile:
+                                line = line.strip()
+                                if (line.startswith("#") and
+                                    not line.startswith("#!")):
```

Comment:
perhaps these lines deserve the `black`-style treatment?

Reply:
Sadly, the `black`-style treatment makes this WAY worse in context. Unless we apply `black` to the entire file, not sure its worth it here.